lib/get_issues.py

```diff
@@ -93,5 +125,5 @@ def query_jira(jira_domain_url, jql_query, credentials_file):
 
 
 if __name__ == "__main__":
-    return_list_issues = search_issues()
-    print(return_list_issues)
+    return_issues = search_issues()
+    print(json.dumps(return_issues, indent=4))
```

Comment:
I know its easier to look at, but I would remove the indent so everything is in one line and compacted so it is smaller in size.

```suggestion
    print(json.dumps(return_issues))
```